src/plugins/orangehrmLeavePlugin/config/routes.yaml

```
@@ -207,6 +207,13 @@ apiv2_leave_reports_data:
   defaults:
     _api: OrangeHRM\Leave\Api\LeaveReportDataAPI
 
+apiv2_leave_employee_leave:
+  path: /api/v2/leave/employees/{empNumber}/leaves
```

```suggestion
  path: /api/v2/leave/employees/leaves
```
:thinking: will go like this, we can utilize for my attendance also